### ISSUE TITLE
Erro provedor ISSNet

### ISSUE THREAD

**Issue by Transis-Felipe** [ACBrNet/ACBr.Net.NFSe#181]
Tentei gerar um xml do provedor ISSNet e deu erro no codigo do serviço.

![image](https://user-images.githubusercontent.com/4941173/129723171-29f1c57b-b4de-4629-b62c-14b415875f55.png)

**Comment by rftd** [comment]
Este tipo de erro é o formato do tipo de serviços, tem provedores que quer que não passe 01.07 e sim 0107, o demo não é ajustado para todos as possíveis formas.

**Comment by Transis-Pedro** [comment]
Sim, sim. Só deixei registrado caso alguém queira arrumar o demo.
